### PR TITLE
Launch-ish

### PR DESCRIPTION
**Pull request by kauffj**
- Re-wrote content on homepage, contribute, glossary, and more
- New components for Resources page
- Headers for API pages
- Crude, unimplemented, redirection
- Link grid as a component
- Remove sitemap
- Other cleanup / refactoring as I went

### CODE REVIEW

File: app/sass/pages/_api.scss [hunk 1]
```diff
@@ -102,27 +107,14 @@
 }
 
 
-
```
Comment (NetOperatorWibby) [inline app/sass/pages/_api.scss]: Why remove this?

File: app/sass/partials/_feature-links.scss [hunk 2]
```diff
@@ -43,17 +38,23 @@
 }
 
 .feature-link__title {
+  top: 85px;
```
Comment (NetOperatorWibby) [inline app/sass/partials/_feature-links.scss]: It's better to have parameters in alphabetical order for easy searching/glancing, unless there are multiple parameters that go together like margin/padding, width/height, top/left, bottom/right.

File: app/views/api.js [hunk 3]
```diff
@@ -6,18 +6,33 @@
...
+import redirectOr404 from "../modules/redirectOr404";
+import headerBlockchain from "../components/api/header-blockchain";
+import headerSdk from "../components/api/header-sdk";
 
 const fetch = require("make-fetch-happen").defaults({ cacheManager: "./cache" });
 
-//  V A R I A B L E
-
-const apiScripts = "<script>" + fs.readFileSync("./app/components/client/api-scripts.js", "utf-8") + "</script>";
+/*
```
Comment (NetOperatorWibby) [inline app/views/api.js]: Why is this here?

File: app/views/home.js [hunk 4]
```diff
@@ -1,34 +1,8 @@
 "use strict";
 
-
-
-//  P A C K A G E
```
Comment (NetOperatorWibby) [inline app/views/home.js]: Why break convention in a repo? Like I mentioned elsewhere, being able to glance and see where you need to go is super helpful.

File: app/views/home.js [hunk 5]
```diff
@@ -41,26 +15,35 @@ module.exports = exports = () => html`
     </section>
 
     <section class="features">
-      ${featureLinks}
+      ${linkGrid([
```
Comment (NetOperatorWibby) [inline app/views/home.js]: This indentation's off.

File: app/views/api.js [hunk 6]
```diff
@@ -38,21 +53,37 @@ module.exports = exports = state => parseApiFile(state.params.wildcard).then(res
...
+}).catch(() => {
+  redirectOr404(state.href);
 });
 
 
 
+
 //  H E L P E R S
 
+function createApiHeader(slug) {
```
Comment (NetOperatorWibby) [inline app/views/api.js]: In master, `lbry.tech/api` defaults to displaying one of these pages. In this branch, that page is scrunched. There should be a redirect for this use case if a default API page isn't going to be used.

File: app/sass/pages/_api.scss [hunk 7]
```diff
@@ -102,27 +107,14 @@
...
- */
-
 .api__content {
-  display: grid;
   float: right;
-  grid-template-columns: repeat(auto-fit, minmax(50%, 1fr));
   width: calc(100% - 200px);
+}
+
+.api__documentation {
```
Comment (NetOperatorWibby) [inline app/sass/pages/_api.scss]: There needs to be a top border here, preferably using the `:before` pseudo-selector so that it sits flush with the background of the first `.api__content__example`.

File: app/sass/pages/_api.scss (hunk 1 above)
Comment (kauffj) [reply app/sass/pages/_api.scss]: probably not removed with intent and was a result of copying/pasting/editing other CSS

I'm a big fan of deterministic formatting (e.g. Prettier) so that these issues never come up

File: app/sass/partials/_feature-links.scss (hunk 2 above)
Comment (kauffj) [reply app/sass/partials/_feature-links.scss]: top goes with `postion: relative`

File: app/views/api.js (hunk 3 above)
Comment (kauffj) [reply app/views/api.js]: this was left over from a conflicted rebased, removed

File: app/views/home.js (hunk 4 above)
Comment (kauffj) [reply app/views/home.js]: every JS file has import or require statements at the top, it didn't seem necessary to have a comment explaining that that's what they are

File: app/views/home.js (hunk 5 above)
Comment (kauffj) [reply app/views/home.js]: fixed

File: app/sass/pages/_api.scss (hunk 7 above)
Comment (kauffj) [reply app/sass/pages/_api.scss]: good call, I think `/api` should just send people to `/resources`. I've updated it as such.

**Comment by kauffj** [comment]
@NetOperatorWibby addressed above, but `snazzy` complained about the spacing you desired on `home.js`. I filed [this ticket](https://github.com/lbryio/lbry.tech/issues/165) for deterministic formatting. I think it makes sense to just add that feature rather than spend time debating spacing (I care about consistency, but I care little about the specific choices.)

File: app/views/api.js (hunk 6 above)
Comment (kauffj) [reply app/views/api.js]: /api now goes to /resources
